onboarding_project/squad_pantry_app/models.py

```diff
@@ -0,0 +1,48 @@
+from django.db import models
+from datetime import datetime
+from django.contrib.auth.models import User
+from django.core.validators import MinValueValidator
+
+# Create your models here.
+
+
+class Dish(models.Model):
+    dish_name = models.CharField(max_length=256, unique=True)
+    DISH_TYPE = (
+        ('NV', 'Non-Vegetarian'),
```

Comment:
Shouldn't these be in the format
```py
DISH_TYPE = (
    (NON_VEG, 'Non-Vegetarian')
)
```

where `NON_VEG` would be a constant defined somewhere else? This way if we decide to change 'NV' to 'NON-V' later we have to make the change at only one place. Also this way we can write :
```py
dish_type = models.CharField(max_length=2, choices=DISH_TYPE, default=NON_VEG)
```

Reply:
If we have to make the change later on we'll have to make the change at only one place. Isn't it?

But this is making much more sense I'll change it.